docs/components/Header.tsx

```diff
@@ -275,7 +286,7 @@ export function Header() {
           shadow
           css={mq({
             '&&': {
-              display: ['none', null, null, 'inline-flex'],
+              display: ['none', null, 'inline-flex', 'inline-flex'],
```

Comment:
To make it easier on yourself you don't need to declare the later break point:
```suggestion
              display: ['none', null, 'inline-flex'],
```

Reply:
yeah nice 👍